src/test-utils/browser-automation/browser-driver.js

```diff
@@ -18,25 +18,22 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// TODO - use a Log
-import {addColor, COLOR} from '../../lib/utils/color';
-
-const ERR_AUTOMATION = `Browser automation error. Check stack trace.
- Also note that Chrome 64 or higher is required.`;
+import {COLOR} from '../../lib/utils/color';
+import Log from '../../lib/log';
+const log = new Log('render-test');
 
 const DEFAULT_CONFIG = {
   process: './node_modules/.bin/webpack-dev-server',
   parameters: ['--config', 'webpack.config.js'],
+  port: 5000,
   options: {maxBuffer: 5000 * 1024}
 };
 
 const DEFAULT_PUPPETEER_OPTIONS = {
-  headless: false,
-  executablePath: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
```

Comment:
Why is this no longer needed? Should we still keep this as a constant so that we can offer the option of running in Chrome?

Reply:
We can open API in config for switching browsers